compass/landice/mesh.py

```diff
@@ -688,22 +688,22 @@ def build_mali_mesh(self, cell_width, x1, y1, geom_points,
     logger.info('writing grid_converted.nc')
     write_netcdf(dsMesh, 'grid_converted.nc')
     levels = section.get('levels')
-    args = ['create_landice_grid_from_generic_MPAS_grid.py',
+    args = ['create_landice_grid_from_generic_mpas_grid',
```

Comment:
@xylar , to clarify, we could replace all of these changes with function calls, right?  I'm assuming you retained the `check_call` calls to minimize how invasive this PR is.  @trhille and I are on board with that, but just wanted to make sure we understood what's going on.

Reply:
I would have to remind myself of the details.  You should not try to call the entry-point functions because the command-line arguments are still expected for those and they won't work as expected.  But in many cases there are other functions with "proper" argument you could call instead and that might be preferable.

In any case, yes, I tried to change things minimally.

Reply:
Looking at this further:
https://github.com/MPAS-Dev/MPAS-Tools/blob/701c8d97dfa733ec7128f32aff5ddd4f48a14e2c/conda_package/mpas_tools/landice/create.py#L10
The function `create_from_generic_mpas_grid()` as written is not really callable because it assumes that `OptionParser.parse_args()` will return the expected command-line arguments (which `compass` obviously wouldn't do).  Elsewhere in `mpas_tools`, we have a function with proper arguments that can be called directly and then we have entry point function that do arg parsing and then call the other function.  It would be great to move in that direction but I didn't try to do that for you all because it was one step too far for me.

Reply:
@xylar , got it - that makes sense.  I'm happy with changing things in this way for now (here and the companion MPAS-Tools PR).